identity-apps-core/apps/x509-certificate-authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources_pt_PT.properties

```diff
@@ -0,0 +1,55 @@
+#
+# Copyright (c) 2024, WSO2 LLC. (http://www.wso2.com).
+#
+# WSO2 LLC. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+
+# <!-- Start of Common Translations -->
+copyright=© {{currentYear}} WSO2 LLC.
+site.title=WSO2 Identity Server
+powered.by=Distribuído por
+privacy.policy=Política de Privacidade
+terms.of.service=Termos de serviço
+# <!-- End of Common Translations -->
+
+certificateNotFound.error.message=Não foi possível encontrar o certificado x509 no navegador!
+userNotFound.error.message=Não foi possível encontrar o nome de usuário para o atributo do X509Certificate configurado!
+userNamesConflict.error.message=Não foi possível encontrar nenhum certificado pertence a esse usuário!
+userNotFoundInUserStore.error.message=Não foi possível encontrar o usuário do Certificado X509 na loja de usuários!
+unknown.error.code=51007
```

Comment:
Why have we kept a constant in the i18n resource files?

Reply:
I'm not sure why there is a number in the i18n files, but this string existed before this refactoring as well. https://github.com/wso2/identity-apps/blob/826a275070d3684a5c397fa8a11e3ae73c952b7e/identity-apps-core/apps/x509-certificate-authentication-portal/src/main/resources/org/wso2/carbon/identity/application/authentication/endpoint/i18n/Resources.properties#L23